src/MainWindow.vala

```diff
@@ -86,16 +83,25 @@ public class Friends.MainWindow : Hdy.ApplicationWindow {
         individual_grid.attach (individualview_header, 0, 0);
         individual_grid.attach (individual_view, 0, 1);
 
-        var paned = new Gtk.Paned (Gtk.Orientation.HORIZONTAL);
-        paned.pack1 (sidebar_grid, false, false);
-        paned.pack2 (individual_grid, true, false);
+        var paned = new Gtk.Paned (Gtk.Orientation.HORIZONTAL) {
+            start_child = sidebar_grid,
+            resize_start_child = false,
+            shrink_start_child = false,
+            end_child = individual_grid,
+            resize_end_child = true,
+            shrink_end_child = false
+        };
 
-        add (paned);
+        child = paned;
+        titlebar = new Gtk.Label ("") { visible = false };
 
         individual_aggregator = Folks.IndividualAggregator.dup ();
         load_contacts.begin ();
 
         Friends.Application.settings.bind ("pane-position", paned, "position", GLib.SettingsBindFlags.DEFAULT);
+        Friends.Application.settings.bind ("window-maximized", this, "maximized", GLib.SettingsBindFlags.DEFAULT);
+        Friends.Application.settings.bind ("window-width", this, "default-width", GLib.SettingsBindFlags.DEFAULT);
+        Friends.Application.settings.bind ("window-height", this, "default-height", GLib.SettingsBindFlags.DEFAULT);
```

Comment:
See my previous comment. You can remove these if you do it the same way as Music